From eba411c42f6022d56a43da147e4ced714ee89999 Mon Sep 17 00:00:00 2001
From: Leon Bottou <leon@bottou.org>
Date: Thu, 14 Feb 2013 18:52:32 -0500
Subject: [PATCH 061/117] removed gstring cruft

---
 libdjvu/GString.cpp | 13 -------------
 1 file changed, 13 deletions(-)

diff --git a/libdjvu/GString.cpp b/libdjvu/GString.cpp
index 467acb8..caf2415 100644
--- a/libdjvu/GString.cpp
+++ b/libdjvu/GString.cpp
@@ -2145,19 +2145,6 @@ GStringRep::concat(const GP<GStringRep> &s1,const GP<GStringRep> &s2) const
   return retval;
 }
 
-#ifdef WIN32
-static const char *setlocale_win32(void)
-{
-  static const char *locale=setlocale(LC_ALL,0);
-  if(! locale || (locale[0] == 'C' && !locale[1]))
-  {
-    locale=setlocale(LC_ALL,"");
-  }
-  return locale;
-}
-const char *setlocale_win32_var = setlocale_win32();
-#endif
-
 GStringRep::GStringRep(void)
 {
   size=0;
-- 
1.8.4

