From 8fd027ffdcc411751cc1010f312470df8498be23 Mon Sep 17 00:00:00 2001
From: Leon Bottou <leon@bottou.org>
Date: Mon, 15 Apr 2013 22:13:44 -0400
Subject: [PATCH 072/117] handle period in crlf files as well

---
 tools/djvused.cpp | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/tools/djvused.cpp b/tools/djvused.cpp
index c739418..d9259d9 100644
--- a/tools/djvused.cpp
+++ b/tools/djvused.cpp
@@ -396,7 +396,8 @@ get_data_from_file(const char *cmd, ParsingByteStream &pbs, ByteStream &out)
       int c = pbs.get_spaces(true);
       pbs.unget(c);
       char skip[4];
-      char term[4] = "\n.\n";
+      char term0[4] = "\n.\n";
+      char term1[4] = "\r.\r";
       char *s = skip;
       int state = 1;
       while (state < 3) 
@@ -404,7 +405,7 @@ get_data_from_file(const char *cmd, ParsingByteStream &pbs, ByteStream &out)
           c = pbs.get();
           if (c == EOF)
             break;
-          if ( c == term[state] )
+          if ( c == term0[state] || c == term1[state] )
             {
               state += 1;
               *s++ = c;
-- 
1.8.4

