From 7041ac8be4245debc4af462dfe2fbd0da0e5ba60 Mon Sep 17 00:00:00 2001
From: Adrien Nader <adrien@notk.org>
Date: Sun, 9 Feb 2014 23:53:41 +0100
Subject: [PATCH 3/3] Tell qmake to use pkg-config.

---
 qtbase/mkspecs/win32-g++/qmake.conf | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/qtbase/mkspecs/win32-g++/qmake.conf b/qtbase/mkspecs/win32-g++/qmake.conf
index 73bc91f..a8a0066 100644
--- a/qtbase/mkspecs/win32-g++/qmake.conf
+++ b/qtbase/mkspecs/win32-g++/qmake.conf
@@ -111,4 +111,7 @@ QMAKE_STRIP             = $${CROSS_COMPILE}strip
 QMAKE_STRIPFLAGS_LIB   += --strip-unneeded
 QMAKE_OBJCOPY           = $${CROSS_COMPILE}objcopy
 QMAKE_NM                = $${CROSS_COMPILE}nm -P
+
+PKG_CONFIG              = pkg-config
+
 load(qt_config)
-- 
1.8.4

